### PR TITLE
Product and Initiative View/Cards, and Dynamic Contact Footer

### PR DESCRIPTION
**Pull request by rishabhnag1**
sorry for a slightly larger pull request, I just wanted to get caught up to where I was last week

### CODE REVIEW

File: src/helpers/navigation/openLink.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import {Linking,} from 'react-native';
+
+export default (url) =>
+  Linking.canOpenURL(url).then(supported => {
```
Comment (clintonjrobinson) [inline src/helpers/navigation/openLink.js]: Nice.  Good example of reusable code.

File: src/components/ContactFooter.js [hunk 2]
```diff
@@ -2,21 +2,89 @@ import React, { Component } from 'react';
...
+        url: 'https://www.joinlane.com',
+        address: '46 Stewart St, Toronto, ON, Canada, A1B 2C3'
+      }],
+    card: 0
+
+  }
+
   constructor(props) {
     super(props);
+    this.state = {
```
Comment (clintonjrobinson) [inline src/components/ContactFooter.js]: @rishabhnag1 Any specific reason you are moving the data from props to state?

File: src/components/Initiative/InitiativeRow.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    marginBottom: 20
+  },
+  textContainer: {
+    width: 300,
+    marginHorizontal: 10
+  },
+  text: {
+    fontFamily: 'Avenir',
+    fontSize: 18,
+    color: '#A9A9A9',
```
Comment (clintonjrobinson) [inline src/components/Initiative/InitiativeRow.js]: @rishabhnag1 Maybe we should do a branch just to start refactoring styles out.  Looks like this is a common style for text.

File: src/components/Product/ProductRow.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    shadowOpacity: 0.2
+  },
+  textContainer: {
+    width: 300,
+    marginHorizontal: 10
+  },
+  text: {
+    fontFamily: 'Avenir',
+    fontSize: 18,
+    color: '#A9A9A9',
```
Comment (clintonjrobinson) [inline src/components/Product/ProductRow.js]: @rishabhnag1 Same with colors.

File: src/models/ProductModel.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import Model from 'models';
+
+const ProductModel = Model.create(
```
Comment (clintonjrobinson) [inline src/models/ProductModel.js]: Nice use of models.

Comment (clintonjrobinson) [inline src/models/ProductModel.js]: I can show you how to hook up the models to the Component so every time the model updates the component auto-updates.

The really cool part is when you connect the model on the mobile device to the Server API.  So everytime the data on the server changes, the model on the mobile device changes, and the UI changes.  Without having to do anything.

Comment (rishabhnag1) [inline src/models/ProductModel.js]: that sounds incredible, i definitely need to dive deeper into the model repo

File: src/components/ContactFooter.js (hunk 2 above)
Comment (rishabhnag1) [inline src/components/ContactFooter.js]: oh whoops, I set that up assuming i'd need it and then forgot to delete it after, thanks for the catch

File: src/components/Initiative/InitiativeRow.js (hunk 3 above)
Comment (rishabhnag1) [inline src/components/Initiative/InitiativeRow.js]: yeah i'll take that task on as soon as I finish the navigator setup, lots of repeated styles throughout

**Pull request merged**